resources/js/types/global.d.ts

```diff
@@ -1,8 +1,11 @@
 import type { Alpine } from 'alpinejs';
 
+import type { expandableAwards as ExpandableAwards } from '../utils/expandableAwards';
```

Comment:
It would be great if this could be generalized in some way, not including `awards` in the name if feasible? (Those will be just "badges" in the future) We might want to use this in other places too?

Reply:
`badgeGroup`, perhaps?

Reply:
I've changed `expandableAwards` to `badgeGroup`.